variants/XMC1400/config/XMC1400_Arduino_Kit/pins_arduino.h

```diff
@@ -0,0 +1,290 @@
+#define XMC_BOARD           XMC 1400 Kit for Arduino
+
+/* On board LED is ON when digital output is 0, LOW, FALSE, OFF */
+#define  XMC_LED_ON         0
+
+// Following were defines now evaluated by compilation as const variables
+// After definitions of associated mapping arrays
+extern const uint8_t NUM_DIGITAL;
+extern const uint8_t GND;
+extern const uint8_t NUM_PWM4;
+extern const uint8_t NUM_PWM;
+extern const uint8_t NUM_INTERRUPT;
+extern const uint8_t NUM_ANALOG_INPUTS;
+#define NUM_LEDS            3
+#define NUM_SERIAL          1
+#define NUM_TONE_PINS       4
+#define NUM_TASKS_VARIANT   8
+
+// Indicate unit has RTC/Alarm
+#define HAS_RTC             1
+
+// Defines will be either set by ArduinoIDE in the menu or manually
+#ifdef SERIAL_HOSTPC
+// Comment out following line to use Serial on pins (board)
+#define SERIAL_DEBUG    1
+#elif SERIAL_ONBOARD
+// No SERIAL_DEBUG will be defined, kept here for clarity
+#else
+// Define the SERIAL_DEBUG as default setting
+#define SERIAL_DEBUG    1
+#endif
+
+#define PWM4_TIMER_PERIOD (2041U)  // Generate 490Hz @fCCU=1MHz
+
+#define PCLK 96000000u //PCLK can go to this max value
+
+#define PIN_SPI_SS    10
+#define PIN_SPI_MOSI  11
+#define PIN_SPI_MISO  12
+#define PIN_SPI_SCK   13
+
+#define A0   0
+#define A1   1
+#define A2   2
+#define A3   3
+#define A4   4
+#define A5   5
+
+#define AD_AUX_1    6  // AD_AUX
+#define AD_AUX_2    7  // AD_AUX
+#define AD_AUX_3    8  // AD_AUX
+#define AD_AUX_4    9  // AD_AUX
+#define AUX_1       26  // AUX
+
+#define LED1        13
+#define LED2        26
+#define LED3        27
+#define LED_BUILTIN LED1
+
+#define EXT_INTR_0  3
+#define EXT_INTR_1  26
+
+#define digitalPinToInterrupt(p)    ((p) == 3 ? 0 : ((p) == 26 ? 1 : NOT_AN_INTERRUPT))
+
+/* Mapping interrupt handlers. Notice that XMC1400 can have interrupt handlers working in 3 modes, the defines below assumes the mode A.
+   For details refer to assembly file and reference manual.
+*/
+#define USIC0_0_IRQHandler IRQ9_Handler // UART
+#define USIC0_0_IRQn IRQ9_IRQn
+
+#define CCU40_0_IRQHandler IRQ21_Handler // interrupt 1
+#define CCU40_0_IRQn IRQ21_IRQn
+
+#define CCU40_1_IRQHandler IRQ22_Handler // interrupt 0
+#define CCU40_1_IRQn IRQ22_IRQn
+
+#define USIC0_4_IRQHandler IRQ13_Handler // I2C
+#define USIC0_4_IRQn IRQ13_IRQn
+
+#define USIC0_5_IRQHandler IRQ14_Handler // I2C
+#define USIC0_5_IRQn IRQ14_IRQn
+
+#define SCU_1_IRQHandler IRQ1_Handler //RTC
+#define SCU_1_IRQn IRQ1_IRQn
+
+#define ERU0_0_IRQHandler IRQ3_Handler // RESET
+#define ERU0_0_IRQn IRQ3_IRQn
+
+#ifdef ARDUINO_MAIN //index is arduino pin count 
+// Mapping of digital pins and comments
+const XMC_PORT_PIN_t mapping_port_pin[] =
+    {
+    /* 0  */    {XMC_GPIO_PORT1, 2}, // RX                                        P1.2
+    /* 1  */    {XMC_GPIO_PORT1, 3}, // TX                                        P1.3
+    /* 2  */    {XMC_GPIO_PORT0 , 5}, // GPIO / LED2                              P0.5
```

Comment:
According to board manual AND standard UNO Shield R3 this should be int 0 P1.1 according to board and P1.4 according to manual